src/utils/count_target_bytes.ts

```diff
@@ -25,6 +25,8 @@ export const countTargetBytes = (source: net.Socket, target: net.Socket): void =
     source[targetBytesRead] = source[targetBytesRead] || 0;
     source[targets] = source[targets] || new Set();
 
+    source[targets].add(target);
```

Comment:
This is an urgent fix, otherwise stats would be updated only on target socket close.